<!--
 Copyright 2022 Shadow Robot Company Ltd.
 This program is free software: you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the Free
 Software Foundation version 2 of the License.
 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
 more details.
 You should have received a copy of the GNU General Public License along
 with this program. If not, see <http://www.gnu.org/licenses/>.
-->

<launch>
    <arg name="hand_id" default="rh"/>
    <arg name="fingers" default="ff"/>
    <node name="sr_$(arg hand_id)_piezo_feedback" pkg="sr_piezo_feedback" type="sr_piezo_feedback_node.py" output="screen">
        <param name="/hand_id" value="$(arg hand_id)"/>
        <param name="/fingers" value="$(arg fingers)"/>
    </node>
</launch>